process/factory/shard/preProcessorsContainerFactory.go

```diff
@@ -187,25 +190,28 @@ func (ppcm *preProcessorsContainerFactory) Create() (process.PreProcessorsContai
 }
 
 func (ppcm *preProcessorsContainerFactory) createTxPreProcessor() (process.PreProcessor, error) {
-	txPreprocessor, err := preprocess.NewTransactionPreprocessor(
-		ppcm.dataPool.Transactions(),
-		ppcm.store,
-		ppcm.hasher,
-		ppcm.marshalizer,
-		ppcm.txProcessor,
-		ppcm.shardCoordinator,
-		ppcm.accounts,
-		ppcm.requestHandler.RequestTransaction,
-		ppcm.economicsFee,
-		ppcm.gasHandler,
-		ppcm.blockTracker,
-		block.TxBlock,
-		ppcm.pubkeyConverter,
-		ppcm.blockSizeComputation,
-		ppcm.balanceComputation,
-		ppcm.epochNotifier,
-		ppcm.optimizeGasUsedInCrossMiniBlocksEnableEpoch,
-	)
+	args := preprocess.ArgsTransactionPreProcessor{
+		TxDataPool:           ppcm.dataPool.Transactions(),
+		Store:                ppcm.store,
+		Hasher:               ppcm.hasher,
+		Marshalizer:          ppcm.marshalizer,
+		TxProcessor:          ppcm.txProcessor,
+		ShardCoordinator:     ppcm.shardCoordinator,
+		Accounts:             ppcm.accounts,
+		OnRequestTransaction: ppcm.requestHandler.RequestTransaction,
+		EconomicsFee:         ppcm.economicsFee,
+		GasHandler:           ppcm.gasHandler,
+		BlockTracker:         ppcm.blockTracker,
+		BlockType:            block.TxBlock,
+		PubkeyConverter:      ppcm.pubkeyConverter,
+		BlockSizeComputation: ppcm.blockSizeComputation,
+		BalanceComputation:   ppcm.balanceComputation,
+		EpochNotifier:        ppcm.epochNotifier,
+		OptimizeGasUsedInCrossMiniBlocksEnableEpoch: ppcm.optimizeGasUsedInCrossMiniBlocksEnableEpoch,
```

Comment:
align?

Reply:
this is the way gofmt formats it for me, can't find the option to change it.